mcs/class/corlib/corlib.dll.sources

```diff
@@ -1661,3 +1661,7 @@ corefx/SR.cs
 ../../../external/corefx/src/System.Collections.Concurrent/src/System/Collections/Concurrent/OrderablePartitioner.cs
 ../../../external/corefx/src/System.Collections.Concurrent/src/System/Collections/Concurrent/Partitioner.cs
 ../../../external/corefx/src/System.Collections.Concurrent/src/System/Collections/Concurrent/PartitionerStatic.cs
+
+System.Security.Cryptography.X509Certificates/X509CertificateImplApple.cs
+System.Security.Cryptography.X509Certificates/X509Helper.Apple.cs
```

Comment:
The integration of the import is wrong as it switches everyone to mac implementation even for Mono on windows. You need to use `ONLY_APPLETLS` instead in X509Helper.*.cs files